src/main/java/org/kohsuke/github/GHHooks.java

```diff
@@ -77,7 +77,7 @@ public GHHook createHook(String name, Map<String, String> config, Collection<GHE
             }
 
             GHHook hook = root.createRequest()
-                    .method("POST")
+                    .method("PATCH")
```

Comment:
Does `POST` not work?

https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#create-a-repository-webhook